ux/sheet_settings_dockable.go

```diff
@@ -221,6 +222,11 @@ func (d *sheetSettingsDockable) createOptions(content *unison.Panel) {
 			d.settings().ShowIQBasedDamage = d.showIQBasedDamage.State == check.On
 			d.syncSheet(false)
 		})
+	d.UseSkillTrees = d.addCheckBoxWithLink(panel, i18n.Text("Use Skill Trees"), "PU10:1+",
```

Comment:
That link should be to a specific page, not something with a trailing + -- that format is just used for display, not for navigation.